server/comms/comms_test.go

```diff
@@ -329,28 +329,35 @@ func TestClientRequests(t *testing.T) {
 		if err != nil {
 			t.Fatalf("ban route send error: %v", err)
 		}
+		// Banish does Disconnect, but Send is async! Wait, or use SendNow.
+		time.Sleep(50 * time.Millisecond)
 		c.Banish()
```

Comment:
@buck54321 Here's a use case for SendNow.  I hit this bug while testing the send/outHandler refactoring commit, so I decided to put this in a separate commit.